.travis.yml

```diff
@@ -29,7 +29,7 @@ matrix:
           packages:
             - gcc-5
             - g++-5
-      env: DEVITO_ARCH=gcc-5 DEVITO_OPENMP=0 INSTALL_TYPE=conda
+      env: DEVITO_ARCH=gcc-5 DEVITO_OPENMP=0 SEISMIC=examples
```

Comment:
can you leave INSTALL_TYPE=conda ? it'd make things easier to understand

Reply:
Yes